.github/workflows/build.yml

```diff
@@ -36,6 +36,9 @@ jobs:
         with:
           arguments: spotlessApply
 
+      - name: Delete empty files
+        run: find $(git ls-files -m) -size 0 -delete
```

Comment:
Shouldnt this be run before spotless?

Reply:
No. The files are present but contains only `{}`. The spotless rule will delete this, but cannot remove the file. This will find any empty files present after the spotless run, and delete those.

Not the most elegant solution, but it works.